exporter/skywalkingexporter/logrecord_to_logdata.go

```diff
@@ -72,9 +72,6 @@ func resourceToLogData(resource pdata.Resource, logData *logpb.LogData) {
 	}
 
 	attrs.Range(func(k string, v pdata.AttributeValue) bool {
-		if k == conventions.AttributeServiceName || k == conventions.AttributeServiceInstanceID {
-			return true
-		}
```

Comment:
not sure what the context of this is?

Reply:
These two fields were excluded because they were already in service and serviceInstance. But sometimes when using tag to analyze data on the skywalking side, this data is still needed.